firebase-inappmessaging-display/src/main/java/com/google/firebase/inappmessaging/display/FirebaseInAppMessagingDisplay.java

```diff
@@ -205,10 +195,7 @@ public void onActivityStarted(final Activity activity) {
   @Keep
   @Override
   public void onActivityPaused(Activity activity) {
-    // clear all state scoped to activity and dismiss fiam
-    headlessInAppMessaging.clearDisplayListener();
-    imageLoader.cancelTag(activity.getClass());
-    removeDisplayedFiam(activity);
+    unbindFiamFromActivity(activity);
```

Comment:
Similar to bind call, can we limit the unbind to a single place rather than having in onPause and onDestroy?